### PR TITLE
[139] Product Page using HTML

### PR DESCRIPTION
**Pull request by Teecols**
I was able to create a product page for infinix note 5 phone using HTML
IMG_20201201_182517_794
IMG_20201201_182534_009

### CODE REVIEW

File: submissions/tosinKolawole/HTML/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      </figure>
+
+     <!--This section is for product price-->
+      <section>
+          <h2><b>Product Price<b></h2>
+          <article>
+             <p> 
+              The Infinix Note 5 Price in Nigeria is best on Beta Market Online stores and its current price starts at N57,000 naira for the 3GB variety and N65,000 naira for the 4GB variety.    
+             </p>
+
```
Comment (Ireti203) [inline submissions/tosinKolawole/HTML/index.html:25]: You did not close this section.

**Review by Ireti203 — CHANGES_REQUESTED** [review]
You have 2 index.html files, you should remove one.

File: submissions/tosinKolawole/HTML/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+                 <li><strong> Infinix Hot 7</strong></li>
+                 <li><strong> Infinix Note 4 Pro</strong></li>
+           </ol>
+           </div> 
+     </section>    
+         
+   <section>
+          <button>Buy Now</button>      
+   </section>   
+
```
Comment (Ireti203) [inline submissions/tosinKolawole/HTML/index.html:55]: Remove whitespace.

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/tosinKolawole/HTML/index.html (hunk 1 above)
Comment (Teecols) [reply submissions/tosinKolawole/HTML/index.html]: This was the previous index.html file I made, it was supposed to be replaced with the new one. I think I have been able to delete it. Thank you

File: submissions/tosinKolawole/HTML/index.html (hunk 2 above)
Comment (Teecols) [reply submissions/tosinKolawole/HTML/index.html]: I really don't know how to do that yet.. I checked Google and the answers I got are confusing..can you kindly explain?

Comment (Ireti203) [reply submissions/tosinKolawole/HTML/index.html]: I mean the empty line

**Review by Ireti203 — CHANGES_REQUESTED** [review]
The index.html file should be in the HTML folder and not directly in the folder with your name.

**Pull request closed without merging**